### PR TITLE
New role, account exec or manager

### PR DESCRIPTION
**Pull request by lukefrake**

### CODE REVIEW

File: _posts/2016-06-17-account-executive-role.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+layout:         post
+title:          "Recruiting an Account Executive / Manager role in Ashford"
+date:           2016-04-23
+author_id:      "luke_frake"
+description:    "Space Between are looking to recruit a Front-end Developer to join their growing team in Ashford, Kent"
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: wrong job

File: _posts/2016-06-17-account-executive-role.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+We currently looking for a keen, skilled & passionate Account Manager or Executive to join our growing, vibrant web agency in Ashford.
+We would love to find an experienced Account Executive who's ready to grow their skills within the business, whilst learning and sharing with the team in a highly collaborative and friendly environment. The role will be a blend of client relationship management, new business development and helping to grow existing accounts. 
+
+### What will you be doing?
+Ensuring high quality client relationships at every touchpoint
+Working with clients to grow their business with us
+Planning, monitoring and troubleshooting project plans, budgets and delivery
+Working with our consultants on new business opportunities, preparing proposals and generating new business leads
+Project reporting and invoicing
+Quality assurance on projects
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: bit sneaky getting this in their JD?

File: _posts/2016-06-17-account-executive-role.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Location: Ashford, Kent
+
+We currently looking for a keen, skilled & passionate Account Manager or Executive to join our growing, vibrant web agency in Ashford.
+We would love to find an experienced Account Executive who's ready to grow their skills within the business, whilst learning and sharing with the team in a highly collaborative and friendly environment. The role will be a blend of client relationship management, new business development and helping to grow existing accounts. 
+
+### What will you be doing?
+Ensuring high quality client relationships at every touchpoint
+Working with clients to grow their business with us
+Planning, monitoring and troubleshooting project plans, budgets and delivery
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md:21]: sounds like a PM to me

File: _posts/2016-06-17-account-executive-role.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Working with our consultants on new business opportunities, preparing proposals and generating new business leads
+Project reporting and invoicing
+Quality assurance on projects 
+
+### We're looking for
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md:26]: some sales?

File: _posts/2016-06-17-account-executive-role.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+An account executive or manager to be based in our Ashford office, but happy to have some travel across Kent & London as required to meet with clients.
+Experience of account management - we're open to meeting people with at least six months experience in account management from another web agency. This could mean you're an account exec looking for the next step up in your career, or an experienced account manager wanting to work in a new business in an exciting phase of growth.
+Someone who shares passion for high quality digital work and has a strong collaborative and energetic ethos.
+
+### In return, you'll get...
+R&D time every fortnight<br/>
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md:35]: for non-devs?

File: _posts/2016-06-17-account-executive-role.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+A fun and supportive working environment<br/>
+Your own Macbook Pro<br/>
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: air?

File: _posts/2016-06-17-account-executive-role.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
+---
+layout:         post
+title:          "Recruiting an Account Executive / Manager role in Ashford"
+date:           2016-04-23
+author_id:      "luke_frake"
+description:    "Space Between are looking to recruit a Account Executive / Manger to join their growing team in Ashford, Kent"
+categories:     space between
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: jobs?

Comment (lukefrake) [inline _posts/2016-06-17-account-executive-role.md]: These don't get used

File: _posts/2016-06-17-account-executive-role.md (hunk 5 above)
Comment (lukefrake) [inline _posts/2016-06-17-account-executive-role.md:35]: 👍

File: _posts/2016-06-17-account-executive-role.md (hunk 4 above)
Comment (lukefrake) [inline _posts/2016-06-17-account-executive-role.md:26]: Addded

File: _posts/2016-06-17-account-executive-role.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+Working with clients to grow their business with us
+Planning, monitoring and troubleshooting project plans, budgets and delivery
+Working with our consultants on new business opportunities, preparing proposals and generating new business leads
+Project reporting and invoicing
+Quality assurance on projects 
+
+### We're looking for
+An account executive or manager to be based in our Ashford office, but happy to have some travel across Kent & London as required to meet with clients.
+Experience of account management - we're open to meeting people with at least six months experience in account management from another web agency. This could mean you're an account exec looking for the next step up in your career, or an experienced account manager wanting to work in a new business in an exciting phase of growth.
+We want people with a proven track records working in sales, experience meeting new people and making great connections.
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: record -s

File: _posts/2016-06-17-account-executive-role.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+Someone who shares passion for high quality digital work and has a strong collaborative and energetic ethos.
+
+
+### In return, you'll get...
+Negotiable commission<br/>
```
Comment (rafriki) [inline _posts/2016-06-17-account-executive-role.md]: Attractive commission structure

**Pull request merged**